.github/CODEOWNERS

```diff
@@ -10,4 +10,4 @@
 # review when someone opens a pull request.
 *       @amundsen-io/amundsen-committers
 
-*.py    @feng-tao @jinhyukchang @allisonsuarez @dikshathakur3119 @verdan @bolkedebruin @mgorsk1
```

Comment:
I know Diksha left Lyft, but she's still an Amundsen committer. She's welcome to remove herself if she no longer wants to be associated with the project, and we'll eventually remove any inactive maintainers, but I think this isn't needed quite yet (and should be done in a separate PR anyhow)

Reply:
I don't believe she had intended to stay involved in the project, so the cleanup made sense. But I am happy to undo as well.

Reply:
@feng-tao ah, you're right, I remembered there being a default demotion to emeritus status due to inactivity in the governance, but that doesn't exist.